common/app/views/fragments/immersiveVideo.scala.html

```diff
@@ -2,8 +2,8 @@
 @import views.support.Video640
 
 @(page: ContentPage)(implicit request: RequestHeader)
-<div class="explore-video">
```

Comment:
This file is used for photo essays as well, but I don't think this class in necessary anymore.